docs/standard/commandline/dependency-injection.md

```diff
@@ -13,7 +13,7 @@ ms.topic: how-to
 
 [!INCLUDE [scl-preview](../../../includes/scl-preview.md)]
 
-Use a [custom binder](model-binding.md#model-binding-more-than-16-options-and-arguments) to inject custom types into a command handler.
+Use a [custom binder](model-binding.md#model-binding-more-than-8-options-and-arguments) to inject custom types into a command handler.
```

Comment:
I wonder if "model binding" is the correct term here, since we don't do any kind of binding of rich models. We really just bind parameters to the handler. 

@KathleenDollard @baronfel

Reply:
I like parameter binding.